pkg/operator/ceph/cluster/mon/service.go

```diff
@@ -77,10 +67,6 @@ func (c *Cluster) createService(mon *monConfig) (string, error) {
 		return "", nil
 	}
 
-	// mon endpoint are not actually like, they remain with the mgrs1 format
-	// however it's interesting to show that monitors can be addressed via 2 different ports
-	// in the end the service has msgr1 and msgr2 ports configured so it's not entirely wrong
-	logger.Infof("mon %q endpoint is [v2:%s:%s,v1:%s:%d]", mon.DaemonName, s.Spec.ClusterIP, strconv.Itoa(int(DefaultMsgr2Port)), s.Spec.ClusterIP, mon.Port)
-
+	logger.Infof("mon %q ip is %s", mon.DaemonName, s.Spec.ClusterIP)
```

Comment:
Should we still leave the old info message since msgr1 is still supported?

Reply:
The ports are very predictable so it just didn't seem necessary to log them and complicate the log message. If the connection ports are needed, they can also be retrieved from the service.